Adding unix timestamp to executable name on projects with name that conflicts with items on project root

discollizard opened the pull request:
Good evening. This PR concerns this issue: https://github.com/drogonframework/drogon/issues/2370

The solution I found to avoid conflict between the generated executable file and the various folders and files on the drogon_ctl generated project root folder was to add a `_<unix_timestamp>` to it.

Commit:
updating cmake generation

an-tao commented:
@discollizard thanks so much for your patch, I prefer changing the folder name rather than the executable name, what do u think?

discollizard commented:
I changed the executable name because by changing it, it should never conflict with any of the folders or files; if I changed the name of the test folder only, the conflict would still appear if the project was named something else on the folder (I tested by naming a project "views" and the error still ocurred). 

I'm still open to suggestions, though :pray:
